bridge/client/app/_components/ktb-event-item/ktb-event-item.component.html

```diff
@@ -35,7 +35,7 @@ <h4 class="m-0 mt-1 mb-1" [class.error]="_event.isFaulty()" [textContent]="_even
       </dt-tag-list>
     </div>
   </div>
-  <div *ngIf="showLabels && _event.getFinishedEvent()?.hasLabels()">
+  <div *ngIf="showLabels && _event.getFinishedEvent()?.hasLabels()" uitestid="keptn-event-item-labels">
```

Comment:
Change this to keptn-event-item-evaluationTags to give it more context

Reply:
Done